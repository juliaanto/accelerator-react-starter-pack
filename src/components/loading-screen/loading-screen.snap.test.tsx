import LoadingScreen from './loading-screen';
import {render} from '@testing-library/react';

describe('Component: LoadingScreen', () => {
  test('should render correctly', () => {
    const {container} = render(<LoadingScreen />);
    expect(container).toMatchSnapshot();
  });
});
