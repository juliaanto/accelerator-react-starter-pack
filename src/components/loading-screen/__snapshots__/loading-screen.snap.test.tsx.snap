// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: LoadingScreen should render correctly 1`] = `
<div>
  <div
    class="lds-ring"
  >
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;
